src/components/skills-quiz/SkillsQuizStepper.jsx

```diff
@@ -200,9 +200,19 @@ const SkillsQuizStepper = ({ isStyleAutoSuggest }) => {
           <Container size="lg">
             <Stepper.Step eventKey="skills-search" title="Skills Search">
               <div className="skills-quiz-dropdown my-4">
-                <p>{SKILLS_QUIZ_SEARCH_PAGE_MESSAGE}</p>
+                <p>
+                  <FormattedMessage
+                    id="enterprise.skills.quiz.v1.skills.search.heading"
+                    defaultMessage="Let edX be your guide. We combine the educational expertise of the world's leading institutions with labor market data to find the right course(s) and program(s) to help you reach your learning and professional goals."
```

Comment:
Is it possible to use either `course and program` or `courses and programs`?

Reply:
It is possible but that would change the current appearance in the UI
<img width="1440" alt="image" src="https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/113524403/1340b051-c1f2-4247-99b4-c0f9549aefd0">